examples/connectivity/plot_mne_inverse_connectivity_spectrum.py

```diff
@@ -65,11 +66,10 @@
 fmin, fmax = 5., 40.
 sfreq = raw.info['sfreq']  # the sampling frequency
 
-con, freqs, times, n_epochs, n_tapers = spectral_connectivity(label_ts,
-        method='wpli2_debiased', mode='multitaper', sfreq=sfreq, fmin=fmin,
-        fmax=fmax, mt_adaptive=True, n_jobs=2)
+con, freqs, times, n_epochs, n_tapers = spectral_connectivity(
+    label_ts, method='wpli2_debiased', mode='multitaper', sfreq=sfreq,
+    fmin=fmin, fmax=fmax, mt_adaptive=True, n_jobs=2)
```

Comment:
I appreciate this style a lot, for multi line parameter matching it's the most efficient pep8 compatible idiom.